core-plugins/docs/ErrorCollector-errortransform.md

```diff
@@ -0,0 +1,61 @@
+# Error Collector
+
+
+Description
+-----------
+The ErrorCollector plugin takes errors emitted from the previous stage and flattens them by adding
+the error message, code, and stage to the record and outputting the result.
+
+Use Case
+--------
+The plugin is used when you want to capture errors emitted from another stage and pass them along
+with all the error information flattened into the record. For example, you may want to connect a sink
+to this plugin in order to store and later examine the error records.
+
+Properties
+----------
+**messageField:** The name of the error message field to use in the output schema. Defaults to 'errMsg'.
+If this is not specified, the error message will be dropped.
```

Comment:
in the UI, it has a default value of 'errMsg'. If the user chooses to delete it, the message will be dropped. I can probably make this clearer.